pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -452,13 +452,19 @@ same number of patterns as the number of @racket[val-expr]s.
 ]
 }
 
-@defform[(match/values expr clause clause ...)]{
+@defform[(match/values expr clause* clause* ...)]{
 If @racket[expr] evaluates to @racket[n] values, then match all @racket[n]
-values against the patterns in @racket[clause ...]. Each clause must contain
+values against the patterns in @racket[clause* ...]. Each clause must contain
 exactly @racket[n] patterns. At least one clause is required to determine how
 many values to expect from @racket[expr].
+
+@examples[#:eval match-eval
+(match/values (values 1 2 3)
+ [(a (? number? b) (? odd? c)) (+ a b c)])
```

Comment:
Nit: indent the clauses with one more space. `match*`s above are wrong too, so feel free to fix those too if you want.

Reply:
nice catch!